cumulusci/tasks/bulkdata/step.py

```diff
@@ -17,6 +16,8 @@
 from cumulusci.core.utils import process_bool_arg
 from cumulusci.tasks.bulkdata.utils import iterate_in_chunks
 
+BULK_BATCH_SIZE = 3
```

Comment:
@prescod Did you really intend to change this from 10000 to 3 or is that left over from testing?

Reply:
cc241d743fac9f07aabd9ac527561adcfd1bd950